episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+
+### Create a new user
+Now let's create a new user for the organization we just created. Here are the steps:
+
+1. Click on the `Users` tab in the administration page.
+2. Click on the `Create user` button.
+3. Fill in the details of the new user.
+    * You can assign the user to an organization by selecting it from the `Organization` dropdown. Only one organization can be selected.
+    * You can assign roles to the user by selecting them from the `Roles` dropdown. Here we assign the `Researcher` role to the new user. You can give the user more permissions by assigning multiple roles or select the operation boxes in the `Permissions` section.
+4. Click on the `Submit` button to create the new user.
+
+![Create a new user](fig/create_user.png)
+
+### Create a new collaboration
+A collaboration is a collection of one or more organizations. Let's create a new collaboration between two organizations. Here are the steps:
+
+1. Click on the `Collaborations` tab in the administration page.
+2. Click on the `Create collaboration` button.
+3. Fill in the details of the new collaboration.
+    * For `Encrypted` box, you can select whether the collaboration should be encrypted or not.
+    * You can select the organizations that will participate in the collaboration by selecting them from the `Organizations` dropdown.
+    * By default, we select the `Register nodes` box. This will ensure the nodes of the organizations are registered in the collaboration. If you don't select this box, you will have to register the nodes manually later.
+
+![Create a new collaboration](fig/create_collab_01.png)
+
+4. Click on the `Submit` button to create the new collaboration.
+    * After submitting the form, you will see a dialog box to ask you to download the API key. The API key is used to authenticate the nodes in the collaboration. Later we will see how to reset the API key for a node.
+
+![Download API keys](fig/create_collab_02.png)
+
+You will see a message:
+
+> API key download
+>
+> The API keys have been downloaded.
+>
+> Please distribute each of these keys privately to each of the organizations. Note that they may reset their API key so that no-one but them knows it.
+
+5. After creating the collaboration, you can see the details of the collaboration by clicking on the specific collaboration listed in the `Collaborations` tab.
+    * You will see what organizations are participating in the collaboration.
+    * You will see the nodes of the organizations that are registered in the collaboration. If no nodes are registered, you can register them manually by clicking the `Register missing nodes` button.
+    * You can also see the algorithm store available for the collaboration. You can add a algorithm store for the collaboration by clicking the `Add algorithm store` button.
+    * You can also see the studies of the collaboration. You can add a study by clicking the `Add study` button.
+
+![Collaboration details](fig/create_collab_03.png)
```

Comment:
In my mind this figure shows a very minor bug: there are 2 organizations in the collaboration and 2 nodes, so the 'Register missing nodes' button should not be shown. 
I put this on my list to fix this soon